### PR TITLE
Increase amount of search snippet text displayed

### PR DESCRIPTION
**Pull request by Jaym3s**

### CODE REVIEW

**Comment by Jaym3s** [comment]
@divoxx @brupm @jeyb

**Comment by Jaym3s** [comment]
https://www.pivotaltracker.com/story/show/113457185

**Comment by brupm** [comment]
:+1: no-qa

File: lib/search_snippet.rb [hunk 1]
```diff
@@ -1,12 +1,14 @@
 class SearchSnippet
-  PRE_MATCHED_CHAR_LENGTH = 75
-  POST_MATCHED_CHAR_LENGTH = 75
+  PRE_MATCHED_CHAR_LENGTH = 100
+  POST_MATCHED_CHAR_LENGTH = 100
```
Comment (Jaym3s) [inline lib/search_snippet.rb]: here's the increase

File: lib/search_snippet.rb [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
   OMISSION_INDICATOR = '...'
   attr_accessor :content
 
-  def initialize(input_query, content)
+  def initialize(input_query, content, options = {})
     @input_query = input_query
     @content = content
+    @pre_matched_char_length = options[:pre_matched_char_length] || PRE_MATCHED_CHAR_LENGTH
+    @post_matched_char_length = options[:post_matched_char_length] || POST_MATCHED_CHAR_LENGTH
```
Comment (Jaym3s) [inline lib/search_snippet.rb]: I allowed them to be overridden

File: spec/lib/search_snippet_spec.rb [hunk 3]
```diff
@@ -2,7 +2,15 @@
 require "search_snippet"
 
 RSpec.describe SearchSnippet do
-  subject { SearchSnippet.new(query, content) }
+  subject { SearchSnippet.new(query, content, options) }
```
Comment (Jaym3s) [inline spec/lib/search_snippet_spec.rb]: So I didn't have to change every single assertion.

**Pull request merged**